Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM python:3.9-slim
 WORKDIR /app
-COPY . /app
```

Comment:
why did you move that around

Reply:
It was me, when after this change you don't have to install requirements every time you build, the build will only copy new files and build the image and use cached requirements